src/ui/common/src/components/pages/workflows/components/MetricItem.tsx

```diff
@@ -73,7 +73,7 @@ const MetricItem: React.FC<MetricItemProps> = ({ metrics }) => {
           display="flex"
           key={metrics[i].metricId}
           justifyContent="space-between"
-          height="30px"
```

Comment:
Can you set the height of each row to be 64px please?

We need to make sure that the units that we work with are divisible by 8.

Reply:
If you make font size 12px for the 'Show More' text, then the height of the box should be around 64px. (i'd still set this manually so that we can get a nice round number there.